Sources/Consent.swift

```diff
@@ -163,5 +175,24 @@ public class Consent: NSObject, Extension {
         if preferencesManager.updateDefaults(with: defaultPrefs) {
             shareCurrentConsents(event: event)
         }
+
+        if let forceSync = config[ConsentConstants.SharedState.Configuration.CONSENT_FORCE_SYNC] as? Bool {
+            self.forceSync = forceSync
```

Comment:
Do we have a case who and when we will be using this config for forceSync?

Reply:
Also why do we have an extra time check condition even after adding forceSync flag?

Reply:
The forceSync configuration is used to revert back to the current behavior of sending a Profile request with every Consent update, regardless if any values change. Customers who are changing sandboxes or orgs and need to update Profiles on each will need this configuration. Customers who use `resetIdentities` will most likely need this option as well. 
The time check is used when forceSync is true, and it is a small optimization to ignore duplicate Consent update requests which arrive within 1 second of each other. For example, we have observed with some applications, on launch, they send the same Consent update request repeatedly within a few milliseconds.